pom.xml

```diff
@@ -3,8 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.qudini</groupId>
     <artifactId>qudini-exceptions</artifactId>
-    <packaging>pom</packaging>
-    <version>1.1-SNAPSHOT</version>
```

Comment:
We've already attempted to deploy 1.1 numbers, so should probably stay at this version.
(https://circleci.com/gh/qudini/qudini-exceptions/6)

Reply:
Done